st2actions/st2actions/container/base.py

```diff
@@ -127,6 +128,40 @@ def _do_run(self, runner):
                     (status, result, context) = runner.run(action_params)
                     result = jsonify.try_loads(result)
 
+            LOG.debug('Validating runner output: %s', runner.runner_type.output_schema)
+
+            if runner.runner_type.output_schema.get('unmodeled'):
+                runner.runner_type.output_schema.pop('unmodeled')
+                LOG.warn(
+                    """Deprecation Notice: This runner has previously had unmodeled
+                    output. In StackStorm 3.1 the output will be placed under the
+                    `output` key."""
+                )
+                runner_schema = {
+                    "additionalProperties": True
+                }
+            else:
+                runner_schema = {
+                    "type": "object",
+                    "properties": runner.runner_type.output_schema,
+                    "additionalProperties": False
+                }
+
+            LOG.info("Action Result: %s", result)
```

Comment:
Should be careful with logging whole result under `.info` - it could contain potentially sensitive information and that's why, IIRC, we only log it under DEBUG in some scenarios.

Reply:
Ack